common/utils/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -48,6 +48,19 @@ case class MessageWithContext(message: String, context: java.util.HashMap[String
     resultMap.putAll(mdc.context)
     MessageWithContext(message + mdc.message, resultMap)
   }
+
+  def ++(s: String): MessageWithContext = {
+    MessageWithContext(message + s, context)
+  }
+}
+
+object MDC {
+
+  implicit class StringImprovements(val s: String) {
+    def ++(mdc: MessageWithContext): MessageWithContext = {
```

Comment:
The method name cannot be called `+` and does not take effect, so it is called `++`

Reply:
In addition, in `String ++ String`, `++` is an `alias` for `contact`, which sounds reasonable.
<img width="784" alt="image" src="https://github.com/apache/spark/assets/15246973/83b60402-e2f5-4a43-91b2-8d25148c514e">